airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4333,6 +4335,25 @@ components:
           type: array
           items:
             "$ref": "#/components/schemas/JobTypeResourceLimit"
+    NormalizationDestinationDefinitionConfig:
```

Comment:
(minor) I think for an API, it might be better to have an always present `normalizationConfig.supported:bool` and then optional `normalizationConfig.otherStuff`.

This way, in typescript you aren't going to need all the existence checks.

Reply:
Thanks! I implemented your suggestion.